packages/core/src/browser/style/sidepanel.css

```diff
@@ -76,21 +83,21 @@
   background: var(--theia-accent-color3);
 }
 
-.p-TabBar.theia-app-sides .p-TabBar-tabIcon,
+.p-TabBar.theia-app-sides .p-TabBar-tabLabel,
 .p-TabBar.theia-app-sides .p-TabBar-tabCloseIcon {
   display: none;
 }
 
-.p-TabBar.theia-app-sides .p-TabBar-tabLabel {
-  position: absolute;
-  min-height: var(--theia-private-sidebar-tab-width);
-  max-height: var(--theia-private-sidebar-tab-width);
-  align-items: flex-start;
-}
-
 .p-TabBar.theia-app-sides .p-TabBar-tabIcon {
-  transform: scale(1.5);
-  max-height: 15px;
+  width: 30px;
+  font-size: 23px;
+  text-align: center;
+  height: 28px;
+  background-repeat: no-repeat;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-size: 26px
```

Comment:
```css

   ... ;
   opacity: 0.6;
}

.p-TabBar.theia-app-sides .p-mod-current .p-TabBar-tabIcon {
    opacity: 1;
}
```
```

Reply:
VS Code also highlights tabs as active when you hover over them.